terraform/main.tf

```diff
@@ -257,3 +538,141 @@ module "gitops_bridge_bootstrap" {
     chart_version = var.addons_versions[0].argocd_chart_version
   }
 }
+
+
+################################################################################
+# Backstage: Bootstrap
+################################################################################
+
+resource "kubernetes_secret" "tls_secret" {
+  count = local.build_backstage ? 1 : 0
+  depends_on = [kubernetes_namespace.backstage_nammespace]
+
+  metadata {
+    name      = "my-tls-secret"
+    namespace = kubernetes_namespace.backstage_nammespace[count.index].metadata[0].name
+  }
+
+  type = "kubernetes.io/tls"
+
+  data = {
+    "tls.crt" = file("tls.crt")  # Adjust the path accordingly
+    "tls.key" = file("tls.key")  # Adjust the path accordingly
+  }
+}
+
+
+
+resource "helm_release" "backstage" {
+  count = local.build_backstage ? 1 : 0
+  depends_on = [ kubernetes_secret.tls_secret ]
+  name       = "backstage"
+  repository = "oci://oowcontainerimages.azurecr.io/helm"
```

Comment:
We need to put the helm chart and the image itself in GHCR associate with this repo.  Ideally, we should use public existing [Backstage helm chart](https://github.com/backstage/charts).

Reply:
Image is here - got to figure out how to link it to repo
https://github.com/orgs/Azure-Samples/packages/container/package/aks-platform-engineering%2Fbackstage

Reply:
I won't be able to do this pre-merge but looks like it wont block anything. Do we want to push this into a central repo? Problem with the Backstage HELM chart is it just deploys a vanilla backstage instance as it states "For Demo Purposes only" without the catalog item to showcase the PE capability.  Do we want to even provide this in the first place or update the guide in the future to just advise customers build the image themselves from scratch every time as part of this repo and manage the image etc themselves. Lets think in the future on this one. https://github.com/backstage/charts